changelog.d/4466.misc

```diff
@@ -0,0 +1 @@
+Don't require sqlite3 when using postgres as the database backend.
```

Comment:
this isn't a change since the last release, so doesn't make sense to list in the changelog. (If it *was* a change since the last release, it would be a bugfix, not a misc.)

It would make more sense to consider it as part of the PR which introduced the breakage (#4306, it turns out). So copy 4306.misc to 4466.misc, and the two will share a line when towncrier builds the changelog.